client/components/editor/TeachingElement.vue

```diff
@@ -24,7 +39,48 @@ export default {
     disabled: { type: Boolean, default: false },
     dragged: { type: Boolean, default: false }
   },
+  data() {
+    return {
+      isFocused: false
+    };
+  },
+  computed: {
+    ...mapGetters('activeUsers', ['activeUsers']),
+    ...mapState('activeUsers', ['sseId']),
+    contentActiveUsers() {
+      const { contentId } = this.element;
+      return orderBy(this.activeUsers.content[contentId], ['created']) || [];
+    },
+    showActiveUsers() {
+      return this.contentActiveUsers.length;
+    },
+    highlight() {
+      if (!this.contentActiveUsers.length) return;
+      const user = first(this.contentActiveUsers);
+      let color;
+      if (user.profileImage) color = user.palette.border;
+      else color = user.palette.background;
+      return `0 0 0 2px ${color}`;
+    },
+    context() {
+      const { courseId, activityId, contentId } = this.element;
+      const { sseId } = this;
+      const routeName = 'tce';
```

Comment:
Can we make in general routeName implicit and just send courseId, activityId and elementId ?